src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationMfaStatusExternalController.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.professionalapi.controller.external;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.professionalapi.controller.SuperController;
+import uk.gov.hmcts.reform.professionalapi.controller.response.MfaStatusResponse;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RequestMapping(
+        path = "/refdata/external/v1/organisations/mfa"
+)
+@RestController
+public class OrganisationMfaStatusExternalController extends SuperController {
+
+    @ApiOperation(
```

Comment:
Please create separate controller i.e. mfa controller

Reply:
This is a new separate controller, called OrganisationMfaStatusExternalController, but I can change the name if it's too confusing.

Reply:
ok thats fine then only please rename to OrgMfaStatusController